src/services/EnvironmentVariables.ts

```diff
@@ -0,0 +1,62 @@
+import { Logger } from "@aws-lambda-powertools/logger";
+import { AppError } from "../utils/AppError";
+import { HttpCodesEnum } from "../models/enums/HttpCodesEnum";
+import { ServicesEnum } from "../models/enums/ServicesEnum";
+
+
+/**
+ * Class to read, store, and return environment variables used by this lambda
+ */
+export class EnvironmentVariables {
```

Comment:
In my [/session PR](https://github.com/alphagov/di-ipv-cri-bav-api/pull/9) I actually didn't add the `EnvironmentVariables` class because I don't see a huge value in using this over `process.env` and found it quite confusing when I joined as I've never seen this kind of thing done in a typescript project before.

Happy to have a chat about it if it's something the team think is valuable, might bring in @ViveakV as well

Reply:
Removed the file as discussed.